peerprep/frontend/src/api/questionApi.ts

```diff
@@ -81,4 +81,16 @@ export const deleteQuestion = async (id: string): Promise<void> => {
   } catch (error) {
     return handleApiError(error);
   }
-};
\ No newline at end of file
+};
+
+export const getQuestionById = async (id: string): Promise<Question> => {
```

Comment:
Great idea to return a Question object asynchronously instead of just the string